pkg/util/lifted/podcondition.go

```diff
@@ -0,0 +1,32 @@
+package lifted
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// +lifted:source=https://github.com/kubernetes/kubernetes/blob/release-1.23/pkg/api/v1/pod/util.go#L340-L347
```

Comment:
Can we lift the code from Kubernetes v1.26.2?

Reply:
> It seems that we already have the same function code here:
> 
> https://github.com/karmada-io/karmada/blob/0bbf03d3238cb19722d1aeb8e3ea3915274ff380/pkg/util/helper/pod.go#L9

Thanks for point out